CONTRIBUTING.md

```diff
@@ -75,8 +75,22 @@ In order to maintain a high software quality standard, we strongly prefer contri
 
 ## Coding Conventions
 
+Here are some rules of thumb in no particular order:
+
+- Do not use wildcard imports (with a few [exceptions](https://github.com/oss-review-toolkit/ort/blob/master/.detekt.yml)).
+- Sort references (like imports, dependencies etc.) strictly alphabetically (ASCII order, i.e. capital letters first),
+  with blank lines between imports from different top-level packages.
+- Group code logically using blank lines, e.g. before and after code blocks enclosed by curly braces, or before return
+  statements.
+- Do not use blank lines directly after opening curly braces.
+- Do not use duplicate blank lines.
```

Comment:
"Do not use duplicate blank lines." could be omit as it is checked automatically.

Reply:
Due to the recent internal discussions I had I'm not limiting myself to non-enforced conventions here. New contributors tend to forget to run `detekt` locally and are surprised that PR checks then fail. Instead, I chose to add the "We aim to have automated checks for these rules [...]" sentence in the end.

Reply:
I believe we should not mention things where we have automated checks for, to save readers time.
Anyhow, I'll leave it up to you.

Reply:
@oheger-bosch @nnobelis @KorSin @MarcelBochtler would it help / have helped you to briefly read about rules that are enforced anyway?

Reply:
IMHO, it is preferable to have a list of coding conventions, no matter whether the rules are enforced or not. If you list only a subset, contributors are likely to produce PRs that fail due to the automatic checks. The time you might have saved during reading is then by far compensated when you have to figure out what went wrong, fix your formatting, and re-push.

Reply:
IMO the detekt outputs are clear enough.
I don't really need a separate entry for the automatically checked detekt rules in the `CONTRIBUTING.md`.

But how about adding that the `detekt` task should be executed?
Even if it is executed in CI, it might be more clear if a new developer knows this and sees it on their terminal that something failed.

Reply:
> But how about adding that the `detekt` task should be executed?

That's already being mentioned (somewhat) at https://github.com/oss-review-toolkit/ort#development. But I can additionally look at rewording my last sentence accordingly.